java/src/jmri/jmrix/AbstractThrottle.java

```diff
@@ -749,8 +749,8 @@ protected static int intSpeed(float speed, int steps) {
 
         if (value < 0) {
             // if we get here, something is wrong and needs to be reported.
-            Exception ex = new Exception("Error calculating speed. Please send logs to the JMRI developers.");
-            log.error(ex.getMessage(), ex);
+            Exception ex = new Exception("Error calculating speed. ");
+            log.error("Please send logs to the JMRI developers.", ex);
```

Comment:
I think it would be good if this line was:
`log.error("Error calculating speed. Please send logs to the JMRI developers.", ex);`

It makes it more clear to why the user should send the log to the JMRI developers.

The exception is on the second line, but in a log with a lot of strange text, it might be confusing to the user if the two error messages are not on the same line.